tests/infra/remote.py

```diff
@@ -231,6 +237,22 @@ def start(self):
         cmd = self._cmd()
         LOG.info("[{}] {}".format(self.hostname, cmd))
         stdin, stdout, stderr = self.client.exec_command(cmd, get_pty=True)
+        _, stdout_, _ = self.proc_client.exec_command(f'ps -ef | pgrep -f "{cmd}"')
```

Comment:
This will probably work quite well because our cmdline tends to be long, but I think the right solution here is to write a pidfile instead. This will also be useful for services wrappers etc.

Reply:
https://github.com/microsoft/CCF/issues/522